vars/publishToArtifactsProdBucket.groovy

```diff
@@ -0,0 +1,35 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+ /**
+ * Library to sign and upload artifacts to artifacts.opensearch.org
+@param Map[assumedRoleName] <required> - IAM role to be assumed for uploading artifacts
+@param Map[source] <required> - Path to yml or artifact file.
+@param Map[destination] <required> - Artifact type in the manifest, [type] is required for signing yml.
+@param Map[signingPlatform] <optional> - The distribution platform for signing. Defaults to linux
+@param Map[sigtype] <optional> - signature type. Defaults to '.sig'
+@param Map[overwrite]<optional> - Allow output artifacts to overwrite the existing artifacts. Defaults to false
```

Comment:
Nit: `overWrite`

Reply:
Hey! So these args are supplied as it is to `sign.sh` so has to be `sigtype` and `overwrite`: https://github.com/opensearch-project/opensearch-build/blob/main/src/sign_workflow/sign_args.py#L45-L57

Reply:
Trying to keep the args as same as possible. Wondering if I should change `signingPlatform` to just `platform`?

Reply:
Fixed!